addons/website_partner/models/res_partner.py

```diff
@@ -9,7 +9,7 @@ class WebsiteResPartner(models.Model):
     _name = 'res.partner'
     _inherit = ['res.partner', 'website.seo.metadata']
 
-    website_description = fields.Html('Website Partner Full Description', strip_style=True, translate=html_translate)
+    website_description = fields.Html('Website Partner Full Description', sanitize_attributes=False, translate=html_translate)
```

Comment:
@bvr-odoo Maybe align this with other frontend field definition: I think there is also the `sanitize_form=False` on other fields like this? And from 16.0, sanitize_overridable, to remember.

Reply:
> sanitize_form=False

@qsm-odoo I had already taken a look at the similar "website_description" fields and it is aligned on them.

Reply:
I don't know which ones you looked at but most have `sanitize_form=False` like I said :wink: There are 2-3 occurrences without it and they are probably mistake as well :grimacing: But ok, let's keep it that way for now, maybe we want it that way in stable, I have to ask the security team about some things too.